.github/.wordlist.txt

```diff
@@ -48,6 +48,8 @@ AIDL
 AlarmCode
 algs
 alloc
+ambd
+ambiot
```

Comment:
Please put the code examples in the .md in code blocks, and then you won't need to add random things like this to the dictionary.